content/docs/operating/getting_started/caduceus.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Caduceus
+sort_rank: 6
+---
+
+# Caduceus
+Currently Caduceus depends on [SNS](https://aws.amazon.com/sns/getting-started/)
+
+# Installation
+-   [RPM](https://xmidt.io/download/#caduceus)
+-   [Binary](https://xmidt.io/download/#caduceus)
+-   Docker (Link TBC)
+
+# Configuration
+Refer to [configuration file](https://github.com/xmidt-org/caduceus/blob/master/caduceus.yaml)
+for how to configure caduceus.
+
+## Fixed
+Under the aws block of the yaml file you will have
+
+```yaml
+aws:
+  accessKey: ""
+  secretKey: ""
+  env: local-dev
+  sns:
+    awsEndpoint: http://goaws:4100
+    region: "us-east-1"
+    topicArn: arn:aws:sns:us-east-1:000000000000:xmidt-local-caduceus
+    urlPath: "/api/v2/aws/sns"
+```
+For local testing you can use [a mock sns instance](https://github.com/p4tin/goaws)
+_**NOTE**_: Do not use this in production. It is untested.
+
+# Validation
+## Test Health
+```bash
+curl -i HOSTNAME:HEALTH_PORT/health
+```
+
+```
+$ curl -i localhost:6001/health
+HTTP/1.1 200 OK
+Content-Type: application/json
+X-Caduceus-Build: development
+X-Caduceus-Flavor: development
+X-Caduceus-Region: local
+X-Caduceus-Server: localhost
+X-Caduceus-Start-Time: 26 Aug 19 18:43 UTC
+Date: Mon, 26 Aug 2019 20:10:28 GMT
+Content-Length: 427
+Connection: close
+
+{"CurrentMemoryUtilizationActive":895574016,"CurrentMemoryUtilizationAlloc":3190504,"CurrentMemoryUtilizationHeapSys":66224128,"MaxMemoryUtilizationActive":946352128,"MaxMemoryUtilizationAlloc":3908816,"MaxMemoryUtilizationHeapSys":66420736,"PayloadsOverHundred":0,"PayloadsOverTenThousand":0,"PayloadsOverThousand":0,"PayloadsOverZero":0,"TotalRequestsDenied":0,"TotalRequestsReceived":0,"TotalRequestsSuccessfullyServiced":0}
+```
+
+
+## Test Webhooks
+
+### Get webhooks
+```bash
+curl -i -H "Authorization: Basic AUTHOKEN" HOSTNAME:PRIMARY_PORT/hooks
+```
+Where HOSTNAME is you DNS record, docker container, or ip address.
```

Comment:
By the time I got to the second page, I no longer cared if these sentences start with "Where" or not. Since they are uniform, I am happy. :smile:

I still think the `you` needs to be `your` though.